src/components/ItemCard.js

```diff
@@ -1,13 +1,34 @@
 import "./ItemCard.css";
+import ChCakeImg from "../images/chocolate.png";
+import { RiFilePaper2Line } from "react-icons/ri";
+import { Link } from "react-router-dom";
 
-export default function ItemCard(props) {
+export default function ItemCard(props, handleDelete) {
+  function handleDelete() {
+    console.log("Clicked");
+  }
```

Comment:
this does not belong into a pull request.

Reply:
agree!